pilot/pkg/networking/util/util.go

```diff
@@ -257,6 +257,12 @@ func SortVirtualHosts(hosts []route.VirtualHost) {
 	})
 }
 
+// IsProxyVersionGE13 checks whether the given Proxy version is greater than or equals 1.3.
+func IsProxyVersionGE13(node *model.Proxy) bool {
```

Comment:
remove this check. See my other PR #15886

Reply:
Thanks! I will merge upstream after you merge the pr.